tests/std/tests/VSO_0000000_vector_algorithms/test.cpp

```diff
@@ -399,7 +450,47 @@ void test_various_containers() {
     test_one_container<list<int>>(); // bidi, not vectorizable
 }
 
+#if _HAS_CXX20
+constexpr bool test_constexpr() {

```

Comment:
This is adding `constexpr` test coverage for most vectorized algorithms (good!) but I observe that `reverse`, `ranges::reverse`, `reverse_copy`, `ranges::reverse_copy`, `swap_ranges`, `ranges::swap_ranges` are missing here.